webapp/components/test/test-runs-query-builder.html

```diff
@@ -23,77 +23,79 @@
   </test-fixture>
 
   <script>
-    // Set up the playground.
-    document.querySelector('test-runs-query-builder').addProduct();
-    document.querySelector('test-runs-query-builder').addProduct();
+    document.addEventListener('WebComponentsReady', () => {
+      // Set up the playground.
+      document.querySelector('test-runs-query-builder').addProduct();
+      document.querySelector('test-runs-query-builder').addProduct();
```

Comment:
I understand these setups need to happen after the component is ready, but do the tests also need to? We don't wait for components ready in other tests, so it seems a bit inconsistent.

Reply:
Eh, I just waited for it either way, but, reverted.